tests/Controllers/RegisterTest.php

```diff
@@ -57,13 +58,15 @@ public function testRegisterActionSuccess()
         $this->seeInDatabase('users', [
             'username' => 'JohnDoe',
         ]);
+
         // User has email/password identity
-        $user = model('UserModel')->where('username', 'JohnDoe')->first(); // @phpstan-ignore-line
+        $user = model(UserModel::class)->where('username', 'JohnDoe')->first();
```

Comment:
This uses the full class name to locate the class which doesn't allow app-level overrides. I think this should be reverted.

Reply:
The full class name does not mean loading the specified class. If you have `App\Models\UserModel`, it will be loaded.
See https://codeigniter4.github.io/CodeIgniter4/concepts/factories.html#example

And this is a test for this package. I don't think app-level overrides does not matter.

Reply:
I had forgotten Factories would do that. Thanks for the reminder. And you're right - it's a test. Ignore me lol